docs/source/user/extensions.rst

```diff
@@ -49,7 +49,15 @@ Using the Extension Manager
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To manage your extensions, you can use the extension manager. By default, the
-manager is disabled, but you can enable it with the following steps:
+manager is disabled. You can enable it by searching **Extension Manager** in the command platte. 
+
+.. figure:: images/extension_manager_enable_manger.png
```

Comment:
Quick typo in the name of the file (file should also be renamed):
```suggestion
.. figure:: images/extension_manager_enable_manager.png
```

Reply:
Sorry for the typo. I'll get it fixed rn.